x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_form.tsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiForm,
+  EuiFormRow,
+  EuiSelect,
+  EuiComboBox,
+  EuiButton,
+  EuiComboBoxOptionOption,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { EngineLogic } from '../../engine';
+import { SearchUILogic } from '../search_ui_logic';
+import { ActiveField } from '../types';
+import { generatePreviwUrl } from '../utils';
+
+const TITLE_FIELD_LABEL = i18n.translate(
```

Comment:
[Optional] If we're not going to inline these copy strings, I think I have a _slight_ preference for moving these in their own `i18n.ts` file or similar (for both this file and `search_ui.tsx`) 🤔 But not a strong preference so feel free to ignore

Reply:
There are quite a few of them, so I think that makes sense.

Reply:
b5384c1afe0934c07d72f0e1e7bec90fc2dea86f